### PR TITLE
Add version strings to Device

### PR DESCRIPTION
**Pull request by rockapot**

### CODE REVIEW

File: proto/delivery/execution.proto [hunk 1]
```diff
@@ -50,5 +50,5 @@ message DeliveryExecution {
 
   // Version of the execution server, for example a commit hash
   // or public repo release version.
-  string ExecutionServerVersion = 7;
+  string server_version = 7;
```
Comment (jinyius) [inline proto/delivery/execution.proto:53]: i think @scottmcmaster made a similar change...

Comment (rockapot) [reply proto/delivery/execution.proto:53]: This is the public generated version, he probably made the change to the internal version.

Comment (danbosnichill) [reply proto/delivery/execution.proto:53]: Yea, he did.

**Review by danbosnichill — APPROVED** [review]

File: proto/delivery/execution.proto (hunk 1 above)
Comment (jinyius) [reply proto/delivery/execution.proto:53]: ah... i saw schema so i didn't see this was the public one.  lgtm!

**Review by jinyius — APPROVED** [review]

**Pull request merged**